### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by ssm-shell** [transcriptionstream/transcriptionstream#3] [Q&A]
Hello. How change language on diarization and transcription?

**Comment by transcriptionstream** [comment]
Greetings [eraterbarnaul](https://github.com/eraterbarnaul)

I will add language as a .env option in the next update, but the language can be set in /ts-gpu/transcribe_example_d.sh [on line 46](https://github.com/transcriptionstream/transcriptionstream/blob/b202306beb87fe43c8c54b22fc927265a0ddbd35/ts-gpu/transcribe_example_d.sh#L46)
                `python3 diarize_parallel.py --batch-size 16 --whisper-model $DIARIZATION_MODEL --language en -a "$audio_file"`
by updating the "en" to the language code of your choice. You will need to either re-run install.sh or rebuild the ts-gpu image after making the change.
